sympy/printing/tests/test_latex.py

```diff
@@ -1116,6 +1116,10 @@ def test_latex_FourierSeries():
     latex_str = \
         r'2 \sin{\left(x \right)} - \sin{\left(2 x \right)} + \frac{2 \sin{\left(3 x \right)}}{3} + \ldots'
     assert latex(fourier_series(x, (x, -pi, pi))) == latex_str
+    latex_str_sin = r'\sin{\left(x \right)}'
+    assert latex(fourier_series(sin(x), (x, -pi, pi))) == latex_str_sin
+    latex_str_4terms = r'\sin{\left(x \right)} + \sin{\left(2 x \right)} + \sin{\left(3 x \right)} + \sin{\left(4 x \right)}'
+    assert latex(fourier_series(sin(x) + sin(2*x) + sin(3*x) + sin(4*x), (x, -pi, pi))) == latex_str_4terms
```

Comment:
Similar tests should be added for pretty. Also something like the `sin(x)**2` example should be added.